### DISCUSSION TITLE
REINTRODUCING HANDSHAKE - A GOOD FAITH PROPOSAL

### DISCUSSION THREAD

**Discussion by NamerTips** [handshake-org/HIPs#72] [Ideas]
![REINTRODUCING HANDSHAKE](https://github.com/user-attachments/assets/6edac05e-ab0e-4b38-94be-4d785334a671)

### **FULL DISCLAIMER:**

**This proposal was drafted with those committed to elevating Handshake in mind. To be clear, elevating Handshake means amplifying the good it can provide in an innovative, strategic and sustainable manner. This includes reaching, and serving, THE PEOPLE.** Note:

- If you're solely focused on advancing your interests, and nothing else, this proposal is **NOT** for you. 

- If you have no interest in reading a critical assessment of the Handshake community dynamic, this proposal is **NOT** for you. 

- If you’re fine with the way things are going, spare yourself a read and go admire the HNS token price and community's reputation.

### **With that out of the way…..**

This proposal is divided into (4) parts: 

- PROTOCOL | CHAIN

- TOKEN

- COMMUNITY

- PEOPLE

If Handshake’s adoption woes are to be cured sooner rather than later, each area MUST be given full consideration. Furthermore, each section outlines the core problems, and proposed solutions, from 'a less-technical standpoint'. Why this approach? It's because every discussion around Handshake being hyper-technical does nothing more than keep it a developer's secret and alienate potential users.

### Let’s dive….

![REINTRODUCING HANDSHAKE PROTOCOL](https://github.com/user-attachments/assets/592c5257-c49f-4bb2-aaed-dd4c2db0c24d)

### **THE PROTOCOL | CHAIN:**

The protocol/chain works. If it didn’t, things like NameCheap acquiring Namebase wouldn’t have occurred along with their subsequent moves thereafter. Granted, this makes NameCheap a competitor to TLD owners in some ways; but that’s an angle to explore later. The point is THE PROTOCOL WORKS. Modifying/forking it too many times could negatively impact its systemic integrity and dilute its purpose. It's like constantly erasing in the same area of a paper only to burn a hole in the paper itself.

The Handshake protocol can sustain one more fork before another signals the chain doesn’t have an inherent resilience and relevance. If another fork is voted upon, it should factor updates beyond just freeing unclaimed token allocations. Considering the chain’s purpose, it’s recommended security updates be THE focus alongside repurposing/redistributing the unclaimed token allocations. 12+ million names are registered now. (Many are held by the same people; but they’re registered nonetheless.) 
### Simply give the chain a security facelift.

![REINTRODUCING HANDSHAKE TOKEN](https://github.com/user-attachments/assets/c62e381e-de89-4414-9b52-ddf2f0a5cf0b)

### **THE TOKEN**

“FREE THE TOKENS”. This is exactly what Handshake deserves. Not for the purpose some in the community have suggested though. Freeing and reallocating the tokens isn’t a big enough incentive for devs to build and users to come. The token isn’t worth much; and a dev would have to view their efforts as being worth the allocated/granted amount to build something cool. Most likely won’t see it this way. A large percentage would probably be sold off and simply hurt the token value even more. Not to mention, haphazardly reallocating the unclaimed tokens could spawn malicious trading behavior on the parts of those with big bags behind the scenes.

So why FREE THE TOKENS?

The answer comes down to **principle** and **accessibility**. The principle is related to preserving Handshake’s broader image. The longer those tokens sit, the more the protocol (and community) seem desperate for them to be claimed. Yes, the allocation plan was a favorable idea…..at the time. Now, the unclaimed tokens look like a misappropriation of funds/value. Honestly, they should’ve been freed alongside the Alexa names; but we’re not going to cry over spilled milk here. Certain people/companies didn’t claim names and tokens. It’s time to convey that certain gifts and gestures have an expiration. FREE THE TOKENS and reposition them for better use.

This is where accessibility comes in…..

At the moment, those millions of HNS tokens are nothing more than a number associated with weaning value. From a financial standpoint, their value can only increase if something, or someone, moves it. There are only so many ways to do this with the tokens locked in technical limbo. Freeing the tokens means making them accessible for whatever the community votes is the best use-case. Until that vote occurs, those tokens should be moved to a multi-sig wallet with at-least 10 individuals in the community as signers. To be clear, no more than 4 should be founding members of the Handshake team. The rest should be from the community.

### #FREETHETOKENS

![REINTRODUCING HANDSHAKE COMMUNITY](https://github.com/user-attachments/assets/e7d876a7-9031-45d3-947b-af729c70304a)

### **THE COMMUNITY:**

So, the Handshake “community” functions more like a group of guys with a common coding interest. There are no formal community standards, no structured approach to gaining consensus and no scalable support of one another. This is exacerbated by a toxic culture of tribalism and unbridled critiques of other naming services/protocols. (Some of which are closer to capturing THE PEOPLE'S attention than Handshake is on a few different fronts.) This has lead to onlookers being convinced HNS is toxic. Obviously it’s not; and that’s thanks to people who CHOOSE to build and advocate. It’s time to give those individuals support, exposure and voting power.

Designating community members as multi-sig wallet signers would be an acknowledgment of their contribution(s) and stewardship. Yes, it’s a responsibility; but it would also convey a renewed sense of leadership, guidance and advisory to the public. In order to start fostering a healthier community reputation, there needs to be a campaign, website and an entertainment-based release to draw attention to the HNS protocol and final token redistribution event. This would potentially attract fresh interest in where Handshake could be headed; and maybe spur new development/investment in the process. All while benefiting THE COMMUNITY and THE PEOPLE.

### An organized community is pivotal to Handshake's adoption.

![REINTRODUCING HANDSHAKE THE PEOPLE](https://github.com/user-attachments/assets/2aeaae67-f698-4596-8faa-22ad5e85bb70)

### **THE PEOPLE**

There’s often talk of Handshake needing a "killer app". For those who understand the art of appealing to consumers, it’s clear great offerings don’t always equate to an increased user-base. Especially in the tech-sector. Real people value things that resonate more than what works. If it works, that’s a plus; but people buy into what connects with them. Even if it’s trash. Handshake doesn’t appeal to the people. Nor does its current community try hard enough to do so. The record reflects “build it and they will come” hasn’t worked. It’s time to pivot to “make it appeal and they’ll embrace it”. This means creating, rewarding and fostering healthy/quality connections. 

'ENGAGEMENT WITH REASON’ is the alpha here. Every front-facing discussion regarding HNS appealing to the people typically comes on the heels of criticizing naming systems with more visibility. Optically, it makes appealing to the people seem like it doesn't really matter to HNS folks. The worst part is more and more people are adopting this outlook on the HNS community; which ultimately hurts the protocol and token. Handshake needs the reintroduction. Not investors. Not devs. Not more VCs. It needs to say "Hi" in a friendly way and follow up with a digital Handshake (gift) of some sort. Talk about a more palatable/marketable greeting. 

### It’s time to ‘reach’ beyond the HNS bubble.

![REINTRODUCING HANDSHAKE SUMMARY](https://github.com/user-attachments/assets/f67074c5-75c9-46e9-bc93-8accd81e2c23)

### **Still not sure what’s being proposed? Here’s a summary:**

- THE PROTOCOL | CHAIN: Fork it once more with an emphasis on doing a security facelift and freeing the unclaimed tokens into a multi-sig wallet. No more forks from there. Let layer 1 be the layer to beat.

- THE TOKEN: FREE THE TOKEN and vote to assign no less than 10 community members (with no more than 4 being founding members of Handshake) to sign-off on token re-allocations in the near future.

- THE COMMUNITY: Launch a dynamic campaign reintroducing Handshake as the first protocol with a community-led token reallocation to/for THE PEOPLE. Use the campaign site to highlight creative content along with the chosen multi-sig wallet signers.

- THE PEOPLE: Appeal to the people with a marketing angle that could be 'FREE THE TOKEN and REACH THE PEOPLE - all as a community'. Provide them with a convenient way to support HNS from there.

### Give THE PEOPLE something to experience, enjoy and be excited about.

![REINTRODUCING HANDSHAKE CONCLUSION](https://github.com/user-attachments/assets/12b91f90-0d7a-48a6-b567-3b59e2eb0a9a)

### **CONCLUSION:** 

Everything proposed is practical and convenient to implement. The forking, token reallocation, multi-sig voting structure and campaign are all necessary. It’s time Handshake go from being “an experiment” to “a building-block” for the future. 

If you're in favor of this proposal, give it a thumbs up, share and express your support for setting up a space to hatch out campaign details. If you don’t agree with any part of this proposal, share your thoughts below and/or come up with something better. Just make sure it truly factors, and resonates with, THE PEOPLE. 

### The future of Handshake depends on it.

![REINTRODUCING HANDSHAKE DISCLAIMER](https://github.com/user-attachments/assets/6a17dc16-b648-43e0-917f-61c23c3a852a)

**_DISCLAIMER: This proposal was submitted by NamerTips via Github in good faith. The viewpoints throughout were provided in relation to helping broaden the appeal, adoption and health of the Handshake protocol, its community and prospective benefactors. Sharing any of the information outlined within this proposal to raise awareness is permitted. NamerTips simply asks that proper attribution be given via link and/or name reference at the time of sharing. This applies to sharing across all mediums including both digital and print. Copying, altering and/or misrepresenting any of the information provided throughout this proposal is strongly prohibited. This proposal does NOT constitute a service arrangement, commitment, agreement, contract or otherwise with parties directly, or indirectly, related to the Handshake protocol and/or HNS token. For questions or comments, please email [info@namer.tips](mailto:info@namer.tips) or reach out via X @NamerTips._** **No AI was used to draft this proposal.** 

### **©Copyright 2024/25**

**Comment by NetOpWibby** [comment]
_Copyright is unnecessary_

---

This is actually a pretty good proposal. You've accuratly pointed out the Handshake community's weak points — perverse incentives/greed and toxicity.

Additionally, the unwillingness to engage with folks and smug technical superiority have done much to damage its reputation. In hindsight, freely giving everyone a prestigous title like "Director of Handshake" to anyone who held even 1 HNS was a mistake. That added a sense of entitlement to the toxicity (why isn't anyone building for us? We're obviously better!) of folks who have done absolutely nothing for the protocol.

I appreciate the time you spent creating this proposal but I'm of the opinion that Handshake is unlikely to fulfill its potential because the community will not change. The vocal minority who are forever focused on token price and the whale lurkers don't see fit to change.

---

Instead, I intend to reboot Handshake to a new chain and community after I launch my current project. Even if Handshake's community woes are fixed, there's still the topic of "killer app(s)." Those come from happy people who are energized, excited, and motivated to build. On every Ethereum chain, excitement is prevalent. Millions are invested into companies on a regular basis. This is not by accident; compare ethereum.org and handshake.org and ask a non-technical person how they'd start contributing.

The solutions are obvious. The work has to be done by everyone, not by a handful and not by only technical folks. A frequent complaint I've seen for years is, "oh, I don't have time" or "I have a life." _Riiiiight_. You don't have time to do anything for Handshake but you can troll for hours? Make it make sense.

_I only commented because [NamerTips brought this](https://twitter.com/NamerTips/status/1842706335193862267) to my attention. My Handshake fatigue remains deep._

**Reply by NamerTips** [reply]
@NetOpWibby Your sentiments and exhaustion align with anyone truly committed to ensuring Handshake serves/benefits THE PEOPLE. You, like a handful of others, have built promising things and advocated from the mountain top. Much respect to you all. 

As for the proposal, it's about no one person or team having to shoulder Handshake's proper reintroduction and promotion to the world. If a sole entity was to do so, THAT would spell the death of Handshake's ethos because monopoly through perception would then apply. NameCheap is already in the ring to convey Handshake to the world as being their product/offering. This is why the healthy part of the community MUST rally now to show that Handshake is 'community before corporate'. (This isn't to imply NameCheap or Richard have a takeover mentality per say.) 

The fact the HNS token and names are separate is a strong anchor-point. After all, the names can be given utility and assigned value at the owner's discretion. Key owners like yourself and Eskimo are builders who merely could use roll-out support that includes a cohesive strategizing and marketing platform. Hence, the platform referenced in the proposal. This creates an easy bridge to connect with the people; and grow an organic userbase. 

As for the whales and their coin interests, they don't really matter here considering the coin isn't accessible to anyone who isn't technical enough to acquire/move it. That's the Achilles heel of being coin-focused in Handshake. It's also the reason price action for the coin is idle. Whales are holding the proverbial egg hoping it hatches. Name holders have the hen that can lay infinite eggs. If the locked tokens can be freed, that'll be a plus. If not, names and owners could still grow forward using some strategizing.

Your fatigue is warranted. It's unfortunate you haven't been supported; but know that Handshake NEEDS Neuenet and @NetOpWibby. Surprisingly, there has been some favorable feedback in the DMs via X regarding this proposal. If we can arrange a space where the details can be ironed out in an organized and progressive manner, would you be open to serving as a valuable perspective? 

![HANDSHAKE NEEDS NEUENET](https://github.com/user-attachments/assets/e22474f7-6873-446e-902b-413b6029d936)

**Reply by NetOpWibby** [reply]
No, I think I made my position well known. Anyone who's been following my progress and work for the past four years understands where I'm coming from. Eskimo and I have been talking to, pleading with, and shouting at folks about the same thing for years. I have to do this with my own kids, I no longer care to do this with alleged adults.

**Comment by pcfreak30** [comment]
Soo... realistically this is just a slightly more specific multisig proposal. The only  thing I would say is `no more hard forks` is unrealistic. im not a fan of BTC's `ossify` philosophy. There is a number of tech that could have benefited BTC in terms of syncing or other aspects that could improve adoption without messing with its economy.

So taking lessons from what I see are mistakes in BTC, I would not advocate for `never forking again`. You fork when its reasonable to, when you have something that can be a high benefit that requires a major upgrade.

Outside of that, this seems to just be a re-hash of similar approaches that I already advocated for, but people seem more interested in burning all the frozen money then using it :shrug:.

**Reply by NamerTips** [reply]
@pcfreak30 Thanks for taking time to review the proposal. It's most appreciated. As for it being "_a slightly more specific multisig proposal_", **the proposal's focal point is launching a platform and campaign that leverages freeing/redistributing the locked tokens along with giving existing builders/projects visibility. All in a way that gives Handshake a fresh image and attracts new interest.** The multisig wallet aspect of things was merely a byproduct of how the tokens would be distributed IF they were released. 

With regard to "_no more hard forks_", the proposal was alluding to doing "forks as necessary". (Hence, the proposal's reference to giving the chain "a security facelift".) Forks are absolutely appropriate in cases where a major tech-upgrade is applicable/needed. However, the sentiment in the community tends to suggest forks are THE answer to non-technical issues related to Handshake's success. All while throwing around the words "integration" and "adoption" in hopes they magically happen as a result of said fork. 

So yeah, the proposal is meant to suggest 'forks as needed and NOT on a whim'. Hopefully this provides some additional clarification on the proposal itself. Thanks again for taking time to check it out. Hopefully you'll be open to sharing your thoughts in a space should there be enough consensus to set one up.
